pkg/cmd/cli/cmd/newapp.go

```diff
@@ -612,6 +612,14 @@ func handleRunError(err error, baseName, commandName, commandPath string) error
 		transformError(err, baseName, commandName, commandPath, groups)
 	}
 	buf := &bytes.Buffer{}
+	if len(config.ArgumentClassificationErrors) > 0 {
+		fmt.Fprintf(buf, "Errors occurred while determining argument types:\n")
+		for _, classErr := range config.ArgumentClassificationErrors {
+			fmt.Fprintf(buf, fmt.Sprintf("\n%s:  %v\n", classErr.Key, classErr.Value))
+		}
+		fmt.Fprint(buf, "\n")
+	}
+	fmt.Fprint(buf, "Errors occurred during resource creation:\n")
```

Comment:
The fact that we are in `handleErrors` at this point already ensures that there is at least one error.  See the if err == nil return snippet at the top.  I believe an additional check is redundant.